Annotation/GraphQLQuery.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @author Thibault Colette <thibaultcolette06@hotmail.fr>
+ * @copyright 2018 Thibault Colette
+ */
+
+namespace Overblog\GraphQLBundle\Annotation;
+
+/**
+ * Annotation for graphql type
+ * Use it if you don't use Doctrine ORM annotation.
+ *
+ * @Annotation
+ * @Target("PROPERTY")
+ */
+class GraphQLQuery
+{
+    /**
+     * @var string
+     */
+    public $method;
+
+    /**
+     * @var array
+     */
+    public $args;
+
+    /**
+     * @var string
+     */
+    public $type;
+
+    /**
+     * @var array
+     */
+    public $input;
+
+    /**
+     * @var string
```

Comment:
Is `$argsBuilder` really a `string` type?

Reply:
Clearly not, array.

Thanks ;)

Reply:
I need more time to fix this one.
I need to study the different way we can use it to make the good implementation in the AnnotationParser when i switched it to array.